test/test_dgmulti_1d.jl

```diff
@@ -9,126 +9,159 @@ EXAMPLES_DIR = joinpath(examples_dir(), "dgmulti_1d")
 
 # Start with a clean environment: remove Trixi.jl output directory if it exists
 outdir = "out"
-isdir(outdir) && rm(outdir, recursive=true)
+isdir(outdir) && rm(outdir, recursive = true)
 
 @testset "DGMulti 1D" begin
-
-  @trixi_testset "elixir_advection_gauss_sbp.jl " begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_gauss_sbp.jl"),
-      cells_per_dimension = (8,),
-      l2 = [2.9953644500009865e-5],
-      linf = [4.467840577382365e-5]
-    )
-    # Ensure that we do not have excessive memory allocations 
-    # (e.g., from type instabilities) 
-    let 
-      t = sol.t[end] 
-      u_ode = sol.u[end] 
-      du_ode = similar(u_ode) 
-      @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000 
+    @trixi_testset "elixir_advection_gauss_sbp.jl " begin
+        @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_gauss_sbp.jl"),
+                            cells_per_dimension=(8,),
+                            l2=[2.9953644500009865e-5],
+                            linf=[4.467840577382365e-5])
```

Comment:
Okay, this is kind of ugly - keyword arguments get no space around `=` for macros. I think we have seen something like this before but I guess we need to live with it.

Reply:
That's similar to https://github.com/domluna/JuliaFormatter.jl/issues/737.